docs/harbor_compatibility_list.md

```diff
@@ -0,0 +1,51 @@
+| ACR | ![ACR](img/replication_adapters/acr.png)|![Y](img/replication_adapters/right.png)|![Y](img/replication_adapters/right.png)| V1.9 | N |
+| AliCR | ![AliCR](img/replication_adapters/ali-cr.png)|![Y](img/replication_adapters/right.png)|![Y](img/replication_adapters/right.png)| V1.9 | N |
+| Helm Hub | ![Helm Hub](img/replication_adapters/helm-hub.png)|![Y](img/replication_adapters/right.png)| N/A | V1.9 | N |
+| Artifactory | ![Artifactory](img/replication_adapters/artifactory.png)|![Y](img/replication_adapters/right.png)| ![Y](img/replication_adapters/right.png) | V1.10 | N |
+| Quay | ![Quay](img/replication_adapters/quay.png)|![Y](img/replication_adapters/right.png)| ![Y](img/replication_adapters/right.png) | V1.10 | N |
+| GitLab Registry | ![GitLab Registry](img/replication_adapters/gitlab.png)|![Y](img/replication_adapters/right.png)| ![Y](img/replication_adapters/right.png) | V1.10 | N |
+
+**Notes**: 
+
+* `Pull` mode is replicating artifacts from specified source registries into Harbor. 
+* `Push` mode is replicating artifacts from Harbor to specified target registries.
+
+## OIDC Adapter
+
+|   |  OIDC Providers | Officially Verified | End User Verified   | Verified in Release |
+|---|-----------------|---------------------|---------------------|-----------------------|
+| Google Identity | ![google identity](img/OIDC/google_identity.png)| ![Y](img/replication_adapters/right.png) |  |V1.9|
+| Dex | ![dex](img/OIDC/dex.png) | ![Y](img/replication_adapters/right.png)| | V1.9 |
+| Ping Identity | ![ping identity](img/OIDC/ping.png) | | ![Y](img/replication_adapters/right.png)| V1.9 |
+| Keycloak | ![Keycloak](img/OIDC/keycloak.png) | ![Y](img/replication_adapters/right.png) | | V1.10 |
+| Auth0 | ![Auth0](img/OIDC/auth0.png) | ![Y](img/replication_adapters/right.png) | | V1.10 |
+
+## Scanner Adapter
+
+|   | Scanners | Providers | Evaluated | As Default | Onboard in Release |
+|---|----------|-----------|-----------|------------|--------------------|
+| [Clair](https://github.com/goharbor/harbor-scanner-clair)    |![Clair](img/scanners/clair.png)| CentOS    |![Y](img/replication_adapters/right.png)|![Y](img/replication_adapters/right.png)| v1.10 |
+| [Anchore](https://github.com/anchore/harbor-scanner-adapter) |![Anchore](img/scanners/anchore.png)   | Anchore    |![Y](img/replication_adapters/right.png)| N | v1.10 |
+| [Trivy](https://github.com/aquasecurity/harbor-scanner-trivy)|![Trivy](img/scanners/trivy.png)| Aqua    |![Y](img/replication_adapters/right.png)| N | v1.10 |
+| [CSP](https://github.com/aquasecurity/harbor-scanner-aqua)   |![Aqua](img/scanners/aqua.png)| Aqua    | N | N | v1.10 |
+| [DoSec](https://github.com/dosec-cn/harbor-scanner/blob/master/README_en.md)|![DoSec](img/scanners/dosec.png)    | DoSec    | N | N | v1.10 |
+
+**Notes:**
+
+* `Evaluated` means the scanner implementation has been officially tested and verified.
```

Comment:
`Evaluated` means that the scanner implementation has been officially tested and verified.